libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -851,6 +866,59 @@ AP_GPS_UBLOX::_parse_gps(void)
         state.horizontal_accuracy = 0;
 #endif
         break;
+
+    case MSG_NAV_RELPOSNED: {
+        Debug("MSG_NAV_RELPOSNED RTK status Received");
+
+        state.rtk_age_ms    = 0xFFFFFFFF;
+
+    	state.rtk_baseline_x_mm = _buffer.rtk.rel_pos_n_cm*10+_buffer.rtk.rel_pos_hp_n_mm;
+    	state.rtk_baseline_y_mm = _buffer.rtk.rel_pos_e_cm*10+_buffer.rtk.rel_pos_hp_e_mm;
+    	state.rtk_baseline_z_mm = _buffer.rtk.rel_pos_d_cm*10+_buffer.rtk.rel_pos_hp_d_mm;
+
+    	int32_t acc_n_mm = _buffer.rtk.acc_n_mm;
+    	int32_t acc_e_mm = _buffer.rtk.acc_e_mm;
+    	int32_t acc_d_mm = _buffer.rtk.acc_d_mm;
+
+    	// rms of error terms
+    	state.rtk_accuracy = safe_sqrt(acc_n_mm*acc_n_mm + acc_e_mm*acc_e_mm + acc_d_mm*acc_d_mm);
```

Comment:
@amilcarlucas accuracy remains unit less on the MAVLink message which is continuing to give headaches here. @rrr6399's solution of providing a 3D cm accuracy (again these mm values are scaled by a factor of 10 before being reported) is reasonable, but there is no way for a GCS/someone reviewing the code to really know what it is...